tsconfig.json

```diff
@@ -17,8 +18,10 @@
     "module": "esnext",
     "moduleResolution": "node",
     "noEmit": false,
+    "noFallthroughCasesInSwitch": true,
```

Comment:
Nice. I don't have the habit of using this, but upon reviewing the docs, it seems like a legitimate value-add.